front/app/modules/commercial/events_widget/index.tsx

```diff
@@ -45,8 +45,8 @@ const configuration: ModuleConfiguration = {
     'app.containers.HomePage.EventsWidget': () => {
       return (
         <RenderOnAllowedAndEnabled>
-          <Box mb="72px">
-            <EventsWidget data-testid="e2e-homepage-events-widget-container" />
+          <Box mb="72px" data-testid="e2e-homepage-events-widget-container">
```

Comment:
On the first attempt, it couldn't find this id, then the user was logged out (I don't know why). Because of that we failed to go to the requested admin page.

Reply:
Interesting. What is the difference between the old and the new code?

Reply:
I didn't investigate. My wild guess is: EventsWidget hadn't loaded yet, so Cypress missed it (racing condition?). The Box and id are there immediately now.